llm_handler/__init___.py

```
@@ -0,0 +1,11 @@
+from .claude import send_to_claude
+from .ollama import send_to_ollama
+
+
+def query_llm(backend="claude", image_path=None, metadata=""):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)